### ISSUE TITLE
1.25.2 ansible-operator is not picking up 8.7 ubi.

### ISSUE THREAD

**Issue by daezaa** [operator-framework/operator-sdk#6183]
#### What did you do?
```
podman image inspect  quay.io/operator-framework/ansible-operator:v1.25.2  | jq .[0].Labels
{
  "architecture": "x86_64",
  "build-date": "2022-10-26T11:06:57",
  "com.redhat.component": "ubi8-container",
  "com.redhat.license_terms": "https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI",
  "description": "The Universal Base Image is designed and engineered to be the base layer for all of your containerized applications, middleware and utilities. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.",
  "distribution-scope": "public",
  "git_commit": "77dd4e2ff165c0317254d55ff01ec1f8d0f1c34a",
  "io.buildah.version": "1.26.2",
  "io.k8s.description": "The Universal Base Image is designed and engineered to be the base layer for all of your containerized applications, middleware and utilities. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.",
  "io.k8s.display-name": "Red Hat Universal Base Image 8",
  "io.openshift.expose-services": "",
  "io.openshift.tags": "base rhel8",
  "maintainer": "Red Hat, Inc.",
  "name": "ubi8",
  "release": "990",
  "summary": "Provides the latest release of Red Hat Universal Base Image 8.",
  "url": "https://access.redhat.com/containers/#/registry.access.redhat.com/ubi8/images/8.6-990",
  "vcs-ref": "f1ee6e37554363ec55e0035aba1a693d3627fdeb",
  "vcs-type": "git",
  "vendor": "Red Hat, Inc.",
  "version": "8.6"
}
```

#### What did you expect to see?
As stated on the [1.25.2 release note](https://github.com/operator-framework/operator-sdk/releases/tag/v1.25.2), the base image should have been picking up v8.7 UBI.

#### What did you see instead? Under which circumstances?
Latest commit to the Dockerfile of ansible-operator is https://github.com/operator-framework/operator-sdk/commit/2bda9e1e767a9007f97422739fa3bbe62cc05a6f which is about a month old.

I think commit of updating base.ansible-operator is missing on ansible-operator dockerfile.

#### Environment

**Operator type:**

/language ansible

#### Possible Solution

Modify  [Dockerfile](https://github.com/operator-framework/operator-sdk/blob/master/images/ansible-operator/Dockerfile) with updated build tag of base.ansible-operator image

**Comment by everettraven** [comment]
Both #6179 and #6180 have been merged and if my understanding is correct the image should be updated automatically within the next day or so. I'll check back on Monday next week and verify whether or not the update has gone through.

**Comment by everettraven** [comment]
Just to follow up, I took a look to see if it has been updated but our automated image refresh job failed this morning. I kicked it off again to see if it was just a flake, but if not I will try to investigate why it is failing so we can get this image rebuilt ASAP

**Comment by everettraven** [comment]
Update: It seems like we would need to go through the process of another patch release to get this taken care of. I have this on my todo list but after some discussion offline with other maintainers we aren't marking this as something that is urgent and rather something that I will get around to when I have some extra time.

**Comment by daezaa** [comment]
@everettraven Thanks for the update, do you have the ETA for next patch release?

**Comment by everettraven** [comment]
@daezaa Due to this week being the week of Thanksgiving in the United States I would give a rough estimate of mid-late next week

**Comment by everettraven** [comment]
@daezaa I just pushed out a new patch release (`v1.25.3`) that updates ansible images to properly pull in the UBI 8.7 image.

I have verified that `quay.io/operator-framework/ansible-operator:(v1.25.3 | v1.25 | latest)` all currently pull in the UBI 8.7 image with `podman image inspect ...`.

Here is the labels output from `podman image inspect quay.io/operator-framework/ansible-operator:latest`:
```json
 {
            "architecture": "x86_64",
            "build-date": "2022-11-01T04:29:09",
            "com.redhat.component": "ubi8-container",
            "com.redhat.license_terms": "https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI",
            "description": "The Universal Base Image is designed and engineered to be the base layer for all of your containerized applications, middleware and utilities. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.",
            "distribution-scope": "public",
            "git_commit": "aa0243440429e60568c27df5c7c22976f2b23a16",
            "io.buildah.version": "1.26.2",
            "io.k8s.description": "The Universal Base Image is designed and engineered to be the base layer for all of your containerized applications, middleware and utilities. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.",
            "io.k8s.display-name": "Red Hat Universal Base Image 8",
            "io.openshift.expose-services": "",
            "io.openshift.tags": "base rhel8",
            "maintainer": "Red Hat, Inc.",
            "name": "ubi8",
            "release": "929",
            "summary": "Provides the latest release of Red Hat Universal Base Image 8.",
            "url": "https://access.redhat.com/containers/#/registry.access.redhat.com/ubi8/images/8.7-929",
            "vcs-ref": "a995512a05037e3b60bbb1bf9fa6e394063131c3",
            "vcs-type": "git",
            "vendor": "Red Hat, Inc.",
           "version": "8.7"
}
```

Since this should now be resolved as of Operator SDK `v1.25.3` I am going to close this issue. Please feel free to reopen if there are any other issues.
